superchain/configs/sepolia-dev-0/superchain.toml

```diff
@@ -10,6 +10,7 @@ fjord_time = 1715961600   # Fri May 17 2024 16:00:00 UTC
 granite_time = 1723046400 # Wed Aug 7 16:00:00 UTC 2024
 holocene_time = 1731682800 # Fri Nov 15 15:00:00 UTC 2024
 pectra_blob_schedule_time = 1741687200 # Tue Mar 11 10:00:00 UTC 2025
+isthmus_time = 1744300800 # Thu Apr 10 16:00:00 UTC 2025
```

Comment:
Unrelated to this PR, but why is this timestamp copied over into the member configs (`base-devnet-0` + `oplabs-devnet-0`)? Shouldn't it just be in one spot?

Reply:
The mental model is that this is an entirely different superchain. The registry has:
* `sepolia-dev-0`,
* `sepolia`, and
* `mainnet` 
superchains. Each has a single set of hardfork activation times (chains in that superchain can opt into inheriting the superchain-wide activation times).

https://github.com/ethereum-optimism/superchain-registry/blob/main/docs/hardfork-activation-inheritance.md